pkgs/development/python-modules/dbt/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchPypi, buildPythonPackage, autopep8 }:
+
+buildPythonPackage rec {
+  pname = "dbt";
+  version = "0.19.0";
+
+  propagatedBuildInputs = [ autopep8 ];
```

Comment:
This package should [depend](https://github.com/dbt-labs/dbt-core/blob/main/setup.py#L24-L26) on `dbt-core`, `dbt-postgres`, `dbt-redshift`, `dbt-snowflake` and `dbt-bigquery`. `dbt` is a meta-package nowadays.

The `setup.py` file also reflect that:

```text
    install_requires=[
        'dbt-core=={}'.format(package_version),
        'dbt-postgres=={}'.format(package_version),
        'dbt-redshift=={}'.format(package_version),
        'dbt-snowflake=={}'.format(package_version),
        'dbt-bigquery=={}'.format(package_version),
    ],
```